master/internal/api_shell.go

```diff
@@ -84,8 +84,8 @@ func (a *apiServer) GetShell(
 		return nil, err
 	}
 
-	if ok, err := user.AuthZProvider.Get().CanAccessNTSCTask(
-		ctx, *curUser, model.UserID(resp.Shell.UserId)); err != nil {
+	if ok, err := command.AuthZProvider.Get().CanGetCommand(
```

Comment:
> we add all authz functions 

hmm, i"m not sure what you mean by all authz functions. do you mean all the calls we'd need to make to authz functions for say shells in this file?
Assuming that's what you're saying I agree I only changed this one because I had to after moving CanAccessNTSC to still have master compile. If we don't move/deprecate that we can't have a full view of what the new commands/authz.* would look like

> 	if ok, err := user.AuthZProvider.Get().CanAccessNTSCTask(

Reply:
yeah that's what I was saying, and I think I understand. I think that should be ok then as long as we fill in the rest of the functions soon.

Reply:
> I only changed this one because I had to after moving CanAccessNTSC to still have master compile. 

I took out this change so non of these api_* files are changed but for notebooks